.github/workflows/ci.yaml

```diff
@@ -8,48 +8,48 @@ on:
 
 jobs:
   test:
-    name: test (${{ matrix.ruby.name }}, ${{ matrix.rails }})
+    name: test (ruby-${{ matrix.ruby }}, ${{ matrix.rails }})
     runs-on: ubuntu-latest
     timeout-minutes: 15
     strategy:
       fail-fast: false
       matrix:
         ruby:
-          - name: ruby-3.0
-            value: "3.0"
-          - name: ruby-3.1
-            value: 3.1
-          - name: ruby-3.2
-            value: 3.2
-          - name: ruby-3.3
-            value: 3.3
+          - "3.0"
+          - "3.1"
+          - "3.2"
+          - "3.3"
```

Comment:
sorry, I've removed this extra complexity because it was difficult to `exclude`ing 3.0/rails_72 and action linter was complaining about a hash value in `ruby`